app/index.js

```diff
@@ -21,7 +21,22 @@ const recompile = _debounce(
       const input = app.get('input');
       const plugins = app.get('selectedPlugins');
 
-      Promise.all(plugins.map(name => pluginsObj[name].import()))
+      Promise.all(
+        plugins.map(name => {
+          const config = Object.assign(
+            {},
+            {
+              browsers:
+                app.get('browsers') ||
+                '> 0.5%, last 2 versions, Firefox ESR, not dead',
```

Comment:
this default config should be externalised to a variable so that we don't have to maintain it both here and in the input placeholder